lib/components/horizontal_news_card.dart

```diff
@@ -11,7 +15,13 @@ class HorizontalNewsCard extends StatelessWidget {
   Widget build(BuildContext context) {
     return InkWell(
       onTap: () {
-        //TODO: implement onTap here
+        Navigator.of(context).pushNamed(
+          AppRouteStrings.articleDetailScreen,
+          arguments: {
+            MapStrings.index: index,
```

Comment:
Make a navigation Model and pass the arguements
Example 
`class ArticleNavDm {
       int index;
       String horizontalCard;
}`